src/wp-content/themes/twentytwentytwo/style.css

```
@@ -17,6 +17,16 @@ Twenty Twenty-Two WordPress Theme, (C) 2021 WordPress.org
 Twenty Twenty-Two is distributed under the terms of the GNU GPL.
 */
 
+/*
+ * IMPORTANT: This file is not normally served directly on the frontend, unless SCRIPT_DEBUG is enabled. For normal page
+ * loads, the `style.min.css` file will be served instead of `style.css` for improved performance. Because of this, if
+ * you make any changes to this file in the Theme File Editor, you will not normally see those changes reflected on the
+ * frontend. You would need to navigate to the theme directory in a terminal and run `npm install && npm run build`.
+ * Because of this, it is not recommended that you use the Theme File Editor to modify this stylesheet. Instead, you
+ * should add the necessary style overrides via "Additional CSS" in the Site Editor. If neither of these are an option,
+ * then you can modify `twentytwentytwo_styles()` in `functions.php` so that it never uses the minified file.
+ */
```

```suggestion
/*
 * IMPORTANT: This file is only served on the frontend when `SCRIPT_DEBUG` is enabled; 
 * in most instances, the `style.min.css` file will be served. It is not recommended that you 
 * use the Theme File Editor to modify this stylesheet. Instead, add the necessary style 
 * overrides via "Additional CSS" in the Site Editor. 
 */
```
There is no specific language standard for CSS, but for PHP [the core standard](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language) are 

> Summaries should be clear, simple, and brief. Avoid describing “why” an element exists, rather, focus on documenting “what” and “when” it does something.

This aims to simplify and make the instructions a bit clearer. Further, build instructions belong in a readme file. Additionally, modifying a theme directly should be discouraged at every opportunity. 


Starting this discussion here, but I think it applies to both style.css files

---

Done in https://github.com/WordPress/wordpress-develop/pull/10081/commits/8ceb867ac17b922a4763d6e46c697c506c909fc7.

---

Props jorbin 😄

---

> Further, build instructions belong in a readme file.

I've added `contributing.txt` files to the two themes in a9e839c, adapted from `twentynineteen`.

---

> Props jorbin 😄

Oh, I think I got confused with the contributor of another PR and accidentally added the wrong co-author to the commit. https://github.com/WordPress/wordpress-develop/commit/8ceb867ac17b922a4763d6e46c697c506c909fc7 😅.